__tests__/utils/env-helpers.test.ts

```
@@ -0,0 +1,45 @@
+
+const originalWindow = global.window
+
+afterEach(() => {
+  Object.assign(process.env, ORIGINAL_ENV)
+  if (originalWindow) {
+    // restore original window if it existed
+    global.window = originalWindow
+  } else {
+    // delete the window added during tests
+    // @ts-ignore
+    delete global.window
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)